web/src/features/fba/components/map/featureStylers.ts

```diff
@@ -60,21 +64,24 @@ export const fireZoneStyler = (
 }
 
 export const getAdvisoryColors = (advisoryThreshold: number, fireZoneArea?: FireZoneArea[]) => {
-  if (isUndefined(fireZoneArea)) {
-    return new Fill({ color: 'rgba(0, 0, 0, 0.0)' })
+  let fill = new Fill({ color: EMPTY_FILL })
+  if (isUndefined(fireZoneArea) || fireZoneArea.length === 0) {
+    return fill
   }
 
-  let fill = new Fill({ color: 'rgba(0, 0, 0, 0.0)' })
   const advisoryThresholdArea = fireZoneArea.find(area => area.threshold == 1)
-  if (advisoryThresholdArea && advisoryThresholdArea.elevated_hfi_percentage > advisoryThreshold) {
+  const warningThresholdArea = fireZoneArea.find(area => area.threshold == 2)
+  const advisoryPercentage = advisoryThresholdArea?.elevated_hfi_percentage || 0
+  const warningPercentage = warningThresholdArea?.elevated_hfi_percentage || 0
+
+  if (advisoryPercentage + warningPercentage > advisoryThreshold) {
```

Comment:
```suggestion
  if ((advisoryPercentage + warningPercentage) > advisoryThreshold) {
```

Reply:
Good idea, but I checked and Prettier just strips out the extra brackets when saving the file in VS Code, so I'll leave it as is.